src/components/Map/CanadaMap.js

```diff
@@ -1,3 +1,26 @@
+/***************************************************************
+ * CanadaMap.js - display component using ncWMS and Leaflet.
+ * Will display up to two map layers, one as shaded color blocks, 
+ * and one as isolines, or either independently, over a tiled 
+ * background. Origin and bounds display all of Canada.
+ * 
+ * Child of (and viewer for) MapController.
+ * 
+ * Props passed to configure shaded color map:
```

Comment:
This isn't a comment on the code necessarily, but why are you naming all of the raster layer with the prefix "scalar". "Scalar" to me implies one single number, and in my mind (and the language we use), we've never called that map a scalar map...

Reply:
ncWMS2 calls the shaded colour map style "scalar," so I was using "scalar" to distinguish that ncWMS raster layer from the other "contour" styled (isoline) ncWMS raster layer. I'm not sure why ncWMS calls that layer style "scalar".

Suggestions for more intuitive-to-us names for the two ncWMS layers? Colours & isolines? Raster & isolines? Chloropeth and isolines?

Reply:
Huh, well if ncWMS calls it that, then I guess that's more sensible. I'd generally vote for raster and isolines, but if you have already used the language of ncWMS, then perhaps we should align with it.